### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by strizr**

### CODE REVIEW

**Comment by strizr** [comment]
TODO: Readme update

File: src/components/visualizer/shared/NetworkToolbar.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <span>center the diagram</span>
+        </v-tooltip>
+        <!--<v-tooltip bottom>
+          <template v-slot:activator="{ on }">
+            <v-btn v-show="buttons.save" class="ma-1" text icon @click="save">
+              <v-icon v-on="on">mdi-file-download-outline</v-icon>
+            </v-btn>
+          </template>
+          <span>save diagram as png</span>
+        </v-tooltip>-->
```
Comment (Simply007) [inline src/components/visualizer/shared/NetworkToolbar.vue:28]: Remove the comment

File: src/components/visualizer/shared/NetworkToolbar.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-card flat dense color="primary" dark class="pa-0 ma-0">
+    <v-row justify="center" align="center" class="pa-0 ma-0">
+      <v-col cols="3" class="pa-0 ma-0 pl-1">
+        <v-tooltip bottom>
+          <template v-slot:activator="{ on }">
+            <v-btn v-show="buttons.fullscreen" class="ma-1" text icon @click="fullscreen">
```
Comment (Simply007) [inline src/components/visualizer/shared/NetworkToolbar.vue:7]: still visible even if on fullscreen

File: src/components/visualizer/shared/NetworkToolbar.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  },
+  methods: {
+    fullscreen() {
+      this.$globalEvents.$emit("fullscreen", true);
+      this.isFullscreen = true;
+    },
+    fit() {
+      this.$emit("fit");
+    },
+    save() {
```
Comment (Simply007) [inline src/components/visualizer/shared/NetworkToolbar.vue:60]: Unused method

File: src/components/visualizer/shared/KontentNetwork.vue [hunk 4]
```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-card outlined class="pa-4">
+    <network-toolbar :buttons="buttons" @fit="fit" @save="save"></network-toolbar>
```
Comment (Simply007) [inline src/components/visualizer/shared/KontentNetwork.vue:3]: I don!T see fit and save buttons in UI - if they are not used, remove them as well as Saving Dialog

File: src/components/visualizer/detail/ModelElements.vue [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  },
+  computed: {
+    filteredElements() {
+      if (this.showAllElements) return this.elements;
+      return this.elements.filter(e => e.type == "LinkedItems");
+    }
+  },
+  methods: {
+    iconFromType(type) {
+      switch (type) {
```
Comment (Simply007) [inline src/components/visualizer/detail/ModelElements.vue:70]: DO we want to add SubPages element?

Comment (Simply007) [reply src/components/visualizer/detail/ModelElements.vue:70]: The cases are probably not probably well defined.

File: src/components/visualizer/overview/SelectedNodeDetail.vue [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+                  class="overline"
+                  expand-icon="mdi-menu-down"
+                >{{selectedNode.links.length}} Linked types</v-expansion-panel-header>
+                <v-expansion-panel-content class="ma-0 pa-0">
+                  <v-list dense>
+                    <v-list-item-group color="primary">
+                      <v-list-item
+                        v-for="(item, i) in linkedTypes"
+                        :key="i"
+                        @click="focus(item.id)"
```
Comment (Simply007) [inline src/components/visualizer/overview/SelectedNodeDetail.vue:64]: I would add at least tooltip, what it does when you click on it.

File: src/components/visualizer/overview/SelectedNodeDetail.vue [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+                            <v-chip
+                              label
+                              color="primary"
+                              outlined
+                              dark
+                              small
+                              v-for="(prop, x) in item.properties"
+                              :key="i + '_' + x"
+                              class="pa-2 mr-2 mb-1"
+                            >{{prop}}</v-chip>
```
Comment (Simply007) [inline src/components/visualizer/overview/SelectedNodeDetail.vue:95]: Looks like clicking does something else - but does also focus.

**Review by Simply007 — CHANGES_REQUESTED** [review]
Just a couple of notes. Overall it is OK and well readable.

File: src/components/visualizer/shared/KontentNetwork.vue (hunk 4 above)
Comment (strizr) [reply src/components/visualizer/shared/KontentNetwork.vue:3]: removing save, fit is visible and usable

**Pull request merged**